.github/workflows/build.yaml

```diff
@@ -0,0 +1,76 @@
+name: Build and Push Consumer Image
+on:
+  push:
+    branches: [ main, release-4.18 ]
+env:
+  VERSION: ${{ github.ref_name == 'main' && 'latest' || github.ref_name == 'release-4.18' && 'release-4.18' }}
```

Comment:
what about just returning the branch name, this way it does not need to be updated later:
```
 VERSION: ${{ github.ref_name == 'main' && 'latest' || github.ref_name }}
```